samples/subsys/fs/littlefs/src/main.c

```diff
@@ -309,6 +309,15 @@ static int littlefs_mount(struct fs_mount_t *mp)
 #endif /* CONFIG_APP_LITTLEFS_STORAGE_FLASH */
 
 #ifdef CONFIG_APP_LITTLEFS_STORAGE_BLK_SDMMC
+
+#if defined(CONFIG_DISK_DRIVER_SDMMC)
+#define DISK_NAME CONFIG_SDMMC_VOLUME_NAME
+#elif IS_ENABLED(CONFIG_DISK_DRIVER_MMC)
+#define DISK_NAME CONFIG_MMC_VOLUME_NAME
+#else
+#error "No disk device defined, is your board supported?"
```

Comment:
What if both drivers are enabled? Or it is not possible?

Reply:
it is definitely possible that we have both drivers or even three (flash map) enabled
but the sample can't work with a few devices at the same time, so we should control what we want to test by a board config or a sample config